.circleci/config.yml

```diff
@@ -757,8 +758,12 @@ jobs:
       - run:
           name: Envoy Integration Tests
           command: |
+            subtests=$(ls -d test/integration/connect/envoy/*/ | xargs -n 1 basename | circleci tests split)
+            echo "Running $(echo $subtests | wc -w) subtests"
+            echo "$subtests"
```

Comment:
Was this just for debugging?

Reply:
No, I think it's helpful to print the list of subtests in case `go test` crashes/panics in the middle of running tests we know whether its run through all the cases or some of them. What do you think?

Reply:
sure